survey-runner.tf

```diff
@@ -398,15 +398,7 @@ module "author" {
       },
       {
         "name": "REACT_APP_API_URL",
-        "value": "${module.author-api.service_address}/graphql"
-      },
-      {
-        "name": "REACT_APP_PUBLISHER_URL",
-        "value": "${module.publisher.service_address}/publish"
-      },
-      {
-        "name": "REACT_APP_GO_LAUNCH_A_SURVEY_URL",
-        "value": "${module.survey-launcher-for-ecs.service_address}/quick-launch"
+        "value": "${module.author-api.service_address}"
```

Comment:
Rather than change this, I think it would be better to introduce a separate launch url environment variable.

```json
{
  "name": "REACT_APP_LAUNCH_URL",
  "value": "${module.author-api.service_address}/launch"
}
```